### ISSUE TITLE
Set compressions

### ISSUE THREAD

**Issue by Kolaer** [s3rius/alarmix#2]
https://github.com/s3rius/alarmix/blob/4392f4673f2e518cde5b53df1529decef496f386/alarmix/daemon/alarm_manager.py#L117-L121

There is a way to rewrite this with set compressions. Should be more readable (subjective, ofc) and possibly faster.

Something like that (typing from phone, so look for errors/typos):

```
 self.alarms[When.auto.value] = {
   alarm_time for alarm_time in self.alarms[When.auto.value]
   if alarm_time >= datetime.now()
 }
```

Also `datetime.now()` calculated for every record in set. I don't think thats a problem, but something inside me screams when I see filtering with dynamic value 🙃

**Comment by s3rius** [comment]
Yeah, you're right.

**Comment by s3rius** [comment]
Fixed in #3

**Comment by Kolaer** [comment]
Nice.
